### PR TITLE
clean up page_mode

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
This is a big change in that it touches a lot of content. @caleybrock could you review as well? Happy to do a walk through kind of review too.

File: pegasus/sites.v3/code.org/public/athletes.haml [hunk 1]
```diff
@@ -63,49 +57,45 @@ video_player: true
...
+  %h2= I18n.t(:sports_athletes_title)
+  %ul
+    %li= I18n.t(:sports_athlete_Draymond)
+    %li= I18n.t(:sports_athlete_Serena)
+    %li= I18n.t(:sports_athlete_Russell)
+    %li= I18n.t(:sports_athlete_Sergio)
+    %li= I18n.t(:sports_athlete_Sue)
+    %li= I18n.t(:sports_athlete_Alana)
+    %li= I18n.t(:sports_athlete_Angelo)
+    %li= "Kobe Bryant, Olympic Gold Medalist and NBA Champion, Los Angeles Lakers"
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/athletes.haml:91]: I will clean up localization on these last minute additions in a separate PR.

File: pegasus/sites.v3/code.org/public/learn/index.haml [hunk 2]
```diff
@@ -97,8 +96,6 @@ social:
   - tutorial[:string_platforms]                     = hoc_s(prefix + "string_platforms")
   - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
   - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
-  - if tutorial[:short_code] == "athlete" && page_mode == "feature-athletes"
-    - tutorial[:tags] = "2016-tutorial"
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/learn/index.haml:101]: Updated in gsheet to not have 'do-not-show' tag

File: pegasus/sites.v3/code.org/views/homepage_video.haml [hunk 3]
```diff
@@ -6,8 +6,8 @@
 
 - video_actions = DB[:cdo_homepage].where(type_s: "action", param1_s: "video", active_s: ["show","hide"], site_s: [request.site,"both"]).all
 - video_actions.each_with_index do |entry, index|
-  - video_code = page_mode == "feature-athletes" ? "ip051U7Rvds" : entry[:param2_s]
-  - download_path = page_mode == "feature-athletes" ? "//videos.code.org/social/athletes-hour-of-code.mp4" : entry[:param3_s]
+  - video_code = entry[:param2_s]
+  - download_path = entry[:param3_s]
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/homepage_video.haml:10]: Both updated in gsheet

File: pegasus/sites.v3/code.org/public/athletes.haml [hunk 4]
```diff
@@ -17,30 +17,24 @@ social:
...
   %h2= I18n.t(:sports_video_title)
-  - url = "https://www.youtube.com/watch?v=#{yt_code}"
+  - url = "https://www.youtube.com/watch?v=ip051U7Rvds"
   - facebook = {u: url}
   - twitter = {url: url, related: 'codeorg', text: "#{I18n.t(:sports_og_description)} @codeorg"}
-  =view :display_video_thumbnail, id: 'athletes_video', video_code: yt_code, play_button: 'center', facebook: facebook, twitter: twitter, download_path: download_link, letterbox: 'false'
-  - if page_mode == "feature-athletes"
-    %a{href: "#featured-athletes"}
-      = I18n.t(:sports_featured_athletes)
+  =view :display_video_thumbnail, id: 'athletes_video', video_code: 'ip051U7Rvds', play_button: 'center', facebook: facebook, twitter: twitter, download_path: '//videos.code.org/social/athletes-hour-of-code.mp4', letterbox: 'false'
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/athletes.haml:35]: this is the right download link now right?

File: pegasus/sites.v3/code.org/public/athletes.haml (hunk 1 above)
Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/athletes.haml:91]: 👍

File: pegasus/sites.v3/code.org/public/athletes.haml [hunk 5]
```diff
@@ -63,49 +57,45 @@ video_player: true
...
-    - videos << { id: 'real_madrid',     code: 'EmJQKAR-78g', text: I18n.t(:video_title_real_madrid),     duration: I18n.t(:video_1min)}
-    - videos << { id: 'marco_belinelli', code: 'Zh6E99aySI4', text: I18n.t(:video_title_marco_belinelli), duration: I18n.t(:video_2min)}
-    - videos << { id: 'chris_bosh',      code: 'kx3DxBbweGY', text: I18n.t(:video_title_chris_bosh),      duration: I18n.t(:video_1min), download: "//videos.code.org/social/athlete-chris-bosh.mp4"}
+  - videos = []
+  - videos << { id: 'draymond_green',  code: 'uJZyqeiAtlA', text: I18n.t(:video_title_draymond_green),  duration: I18n.t(:video_1min), download: "//videos.code.org/social/athlete-draymond-green.mp4"}
+  - videos << { id: 'russell_okung',   code: '-aGEKVvoyLI', text: I18n.t(:video_title_russell_okung),   duration: I18n.t(:video_2min), download: "//videos.code.org/social/athlete-russell-okung.mp4"}
+  - videos << { id: 'real_madrid',     code: 'EmJQKAR-78g', text: I18n.t(:video_title_real_madrid),     duration: I18n.t(:video_1min)}
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/athletes.haml:63]: no download links for these two?

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 6]
```diff
@@ -29,16 +29,8 @@
...
-    - heroes_arranged.insert(0, {:id=>5,:id_s=>"hero_moana",:type_s=>"widehero",:text1_s=>"",:param1_s=>"50% 50%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/moana.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
-    - heroes_arranged.insert(2, {:id=>4,:id_s=>"hero_minecraft2016",:type_s=>"widehero",:text1_s=>"",:param1_s=>"21% 80%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/minecraft2016.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
-  - elsif page_mode == "feature-minecraft2016"
-    - heroes_arranged.insert(0, {:id=>4,:id_s=>"hero_minecraft2016",:type_s=>"widehero",:text1_s=>"",:param1_s=>"21% 80%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/minecraft2016.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
+  -# Temporary for Athletes Push, always show first
+  - heroes_arranged.insert(0, {:id=>6,:id_s=>"hero_athletes",:type_s=>"widehero",:text1_s=>"",:param1_s=>"25% 22%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/athletes.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml:33]: no need for mc or moana?

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 7]
```diff
@@ -29,16 +29,8 @@
...
-  - elsif page_mode == "feature-moana"
-    - heroes_arranged.insert(0, {:id=>5,:id_s=>"hero_moana",:type_s=>"widehero",:text1_s=>"",:param1_s=>"50% 50%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/moana.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
-    - heroes_arranged.insert(2, {:id=>4,:id_s=>"hero_minecraft2016",:type_s=>"widehero",:text1_s=>"",:param1_s=>"21% 80%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/minecraft2016.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
-  - elsif page_mode == "feature-minecraft2016"
-    - heroes_arranged.insert(0, {:id=>4,:id_s=>"hero_minecraft2016",:type_s=>"widehero",:text1_s=>"",:param1_s=>"21% 80%",:param2_s=>"celeba",:param3_s=>"bottom",:image_t=>"/images/homepage/minecraft2016.jpg",:site_s=>"both",:active_s=>"show",:language_supported_s=>"all"})
+  -# Temporary for Athletes Push, always show first
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml:32]: How long is Athletes Push?

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml [hunk 8]
```diff
@@ -101,8 +100,6 @@ social:
   - tutorial[:string_platforms]                     = hoc_s(prefix + "string_platforms")
   - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
   - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
-  - if tutorial[:short_code] == "athlete" && page_mode == "feature-athletes"
```
Comment (caleybrock) [inline pegasus/sites.v3/hourofcode.com/public/learn/index.haml:104]: uses same updated gsheet right?

**Comment by caleybrock** [comment]
Walking through changes would be great too, but I can look after lunch.

File: pegasus/sites.v3/code.org/public/athletes.haml (hunk 4 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/athletes.haml:35]: Correct, it's the same name but the file was updated in aws. Just checked and downloaded now to confirm it has Kobe.

File: pegasus/sites.v3/code.org/public/athletes.haml (hunk 5 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/athletes.haml:63]: No, they aren't ours.

File: pegasus/sites.v3/code.org/views/homepage_hero.haml (hunk 6 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml:33]: Correct. Now that it's post hoc we're removing hoc highlights from hero. MC remains in hocblock permanently.

File: pegasus/sites.v3/code.org/views/homepage_hero.haml (hunk 7 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml:32]: A couple of months.

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml (hunk 8 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/public/learn/index.haml:104]: Correct.

**Pull request merged**